README.md

```diff
@@ -138,6 +139,12 @@ Type representing falsey values in TypeScript: `null | undefined | false | 0 | '
 
 [⇧ back to top](#table-of-contents)
 
+### mixed
+
+Alias for `unknown`
```

Comment:
I would prefer this description: 
"An arbitrary type that could be anything (Same as `unknown`)"
https://flow.org/en/docs/types/mixed